lib/elixir/src/elixir.erl

```diff
@@ -297,3 +297,11 @@ string_to_quoted(String, StartLine, File, Opts) when is_integer(StartLine), is_b
 
 to_binary(List) when is_list(List) -> elixir_utils:characters_to_binary(List);
 to_binary(Atom) when is_atom(Atom) -> atom_to_binary(Atom, utf8).
+
+handle_parsing_opts(File, Opts) ->
+  put(elixir_parser_file, File),
+  case lists:keyfind(wrap_literals_in_blocks, 1, Opts) of {wrap_literals_in_blocks, WrapLiteralsBool} when is_boolean(WrapLiteralsBool) ->
+      put(wrap_literals_in_blocks, WrapLiteralsBool);
```

Comment:
Please remember to `erase` this here: https://github.com/uohzxela/elixir/blob/0dd46bb82ca97c21fb81ce1a698cf8cb0ac55099/lib/elixir/src/elixir.erl#L282